lib/admin/accounts/account.ex

```diff
@@ -11,4 +12,11 @@ defmodule Admin.Accounts.Account do
 
     timestamps(type: :utc_datetime)
   end
+
+  @doc false
+  def changeset(account, attrs) do
+    account
+    |> cast(attrs, [:name, :email, :type])
+    |> validate_required([:name, :email, :type])
```

Comment:
Perhaps out of the scope of this PR: would it be here we validate strings for `name`, `email`, `type`?

Reply:
Yes it is here that we should apply validation of the changes to an account such as email valiadation, name constraints etc. I did not add them when i created the account struct as I needed to only read them. Might be a good idea !